buy-sell/src/index.js

```diff
@@ -4,13 +4,15 @@ import App from "./App";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Register from "./components/Register";
 import { render } from "react-dom";
+import Login from "./components/Login/Login";
```

Comment:
Login/Login code structure is ambiguous. May be you can put Login component in component directory.